mixbench-hip/CMakeLists.txt

```diff
@@ -2,14 +2,14 @@ cmake_minimum_required(VERSION 3.10)
 
 if(NOT DEFINED HIP_PATH)
     if(NOT DEFINED ENV{HIP_PATH})
-        set(HIP_PATH "/opt/rocm/hip" CACHE PATH "Path to which HIP has been installed")
```

Comment:
Can we leave both lines here? Under my environment the variables are the ones that are considered.

Reply:
in the second commit, set the default value to ENV{HIP_PATH}, the if statement can be deleted